jsdoc-conf.json

```diff
@@ -0,0 +1,20 @@
+{
+    "tags": {
+        "allowUnknownTags": true
+    },
+    "source": {
+        "include": ["cli", "generators"],
+        "exclude": ["myProject/lib/ignore.js"],
```

Comment:
may be ignore should include folders like test, node_modules etc, else lets remove it

Reply:
I agree. 👍 
Will update the PR soon